app/errors.js

```diff
@@ -21,3 +19,8 @@ exports.defaultError = message => {
     message
   };
 };
+
+exports.invalidUser = {
```

Comment:
Maybe the name doesn't quite fits, it would be invalid credentials, don't you think?